rr_rviz_plugins/include/rr_rviz_plugins/SpeedometerPanel.h

```diff
@@ -0,0 +1,53 @@
+#ifndef PROJECT_SPEEDOMETERPANEL_H
+#define PROJECT_SPEEDOMETERPANEL_H
+
+#include <ros/ros.h>
+#include <rviz/panel.h>
+#include <rr_platform/chassis_state.h>
+#include <rr_platform/steering.h>
+#include <QLabel>
+#include <QtGui>
+#include <QPainter>
+#include <math.h>
+#include <QString>
+
+
+
+namespace rr_rviz_plugins {
+
+
+class SpeedometerPanel : public rviz::Panel {
+
+
+
```

Comment:
Just a small note on style, we don't need this much whitespace between class declaration and then starting to populate the class. Ditto throughout the rest of these changes where there are just chunks of whitespace where comments used to be.

Reply:
@chsahit do you still think this needs to be fixed before I approve this PR?